### DISCUSSION TITLE
Got FileNotFoundError when I start manager

### DISCUSSION THREAD

**Discussion by pongvech** [swar/Swar-Chia-Plot-Manager#434] [Q&A]
Why I got FileNotFoundError when I start manager? Here is the error details from manager.log

> Traceback (most recent call last):
>   File "C:\Users\Pun\OneDrive\Documents\GitHub\Swar-Chia-Plot-Manager\stateless-manager.py", line 58, in <module>
>     jobs, running_work, next_job_work, next_log_check = monitor_jobs_to_start(
>   File "C:\Users\Pun\OneDrive\Documents\GitHub\Swar-Chia-Plot-Manager\plotmanager\library\utilities\jobs.py", line 119, in monitor_jobs_to_start
>     job, work = start_work(job=job, chia_location=chia_location, log_directory=log_directory)
>   File "C:\Users\Pun\OneDrive\Documents\GitHub\Swar-Chia-Plot-Manager\plotmanager\library\utilities\jobs.py", line 167, in start_work
>     log_file = open(log_file_path, 'a')
> **FileNotFoundError: [Errno 2] No such file or directory: 'C:\\Users\\Pun\\.chia\\mainnet\\plotter\\10/13_2021-05-17_00_40_06_273050.log'**

Here is what my path looks like in config file. It has some logs in it.

`folder_path: C:\Users\Pun\.chia\mainnet\plotter`

**Comment by swar** [comment]
Whats your job name

**Reply by pongvech** [reply]
That solved the problem. The "/" in the job name is the problem. Thanks you very much.
